### DISCUSSION TITLE
What is the smallest interval supported by Edgex Auto Events?

### DISCUSSION THREAD

**Discussion by vediyappan-villali** [edgexfoundry/community#69] [Q&A]
What is the smallest interval supported by Edgex Auto Events? Is that 1s or 100ms or 1ms? 

I couldn't see any details in edgex foundry documentation:  https://docs.edgexfoundry.org/2.3/design/legacy-requirements/device-service/#autoevents

I believe it was mentioned 100ms in earlier versions (1.1 or 1.3) of documentation. But I couldn't see that info now.

**Comment by cloudxxx8** [comment]
there is no hard limit, becuase it would depend on the hardware capability and device protocol nature.  100ms is just a suggestion.  50ms can still work for most of situations.  If you set 1ns, it will crash your system very quick.
